corehq/apps/user_importer/importer.py

```diff
@@ -329,6 +328,8 @@ def check_modified_user_loc(location_ids, loc_id, assigned_loc_ids):
 
 def get_domain_info(domain, upload_domain, user_specs, domain_info_by_domain, upload_user=None, group_memoizer=None, is_web_upload=False):
     from corehq.apps.users.views.mobile.custom_data_fields import UserFieldsView
+    from corehq.apps.users.views.utils import get_editable_role_choices
```

Comment:
why did this have to move?

Reply:
This one was interesting.
It was only happening when I would run tests,
`REUSE_DB=1 ./manage.py test corehq.apps.user_importer.tests.test_importer`
because of 
```
from corehq.apps.user_importer.importer import (
    create_or_update_users_and_groups,
)
```
in the test file.